src/main/menu.ts

```diff
@@ -212,6 +212,14 @@ export default class MenuBuilder {
           this.mainWindow.webContents.send('initiate-export-project');
         },
       },
+      {
+        id: 'updateTranscriptionAPIKey',
+        label: 'Update Transcription API Key',
+        accelerator: 'CommandOrControl+U',
```

Comment:
I know it’s still a WIP but this shortcut would conflict with https://github.com/chloebrett/mlvet/pull/223

maybe Ctrl+K for “key”?